diff --git a/node_modules/@inertiajs/inertia-vue3/index.d.ts b/node_modules/@inertiajs/inertia-vue3/index.d.ts
index 59651a2..7610241 100644
--- a/node_modules/@inertiajs/inertia-vue3/index.d.ts
+++ b/node_modules/@inertiajs/inertia-vue3/index.d.ts
@@ -60,12 +60,12 @@ interface InertiaFormProps<TForm> {
   transform(callback: (data: TForm) => object): this
   reset(...fields: (keyof TForm)[]): this
   clearErrors(...fields: (keyof TForm)[]): this
-  submit(method: string, url: string, options?: Inertia.VisitOptions): void
-  get(url: string, options?: Inertia.VisitOptions): void
-  post(url: string, options?: Inertia.VisitOptions): void
-  put(url: string, options?: Inertia.VisitOptions): void
-  patch(url: string, options?: Inertia.VisitOptions): void
-  delete(url: string, options?: Inertia.VisitOptions): void
+  submit(method: string, url: string, options?: Partial<Inertia.VisitOptions>): void
+  get(url: string, options?: Partial<Inertia.VisitOptions>): void
+  post(url: string, options?: Partial<Inertia.VisitOptions>): void
+  put(url: string, options?: Partial<Inertia.VisitOptions>): void
+  patch(url: string, options?: Partial<Inertia.VisitOptions>): void
+  delete(url: string, options?: Partial<Inertia.VisitOptions>): void
   cancel(): void
 }
 
